README.md

```diff
@@ -73,3 +73,21 @@ After the `bios` files are installed, you can build using `cargo`
 ```sh
 cargo build --workspace --examples
 ```
+
+### Customizing the build
+
+You can customize the build by activating certain feature by using the `--features` flags (comma separated)
+
+| Name                         | Description                                      | Require                                                       |
+| ---------------------------- | ------------------------------------------------ | ------------------------------------------------------------- |
+| time_frame                   | log the time taken to generate a frame           |                                                               |
+| gb-lcd/debug_render          | debug the VRAM and OAM with a GUI                |                                                               |
+| debug_render                 | alias of `gb-lcd/debug_render`                   |                                                               |
```

Comment:
`debug_render`: enable the debug tool for the ppu in the ui.
`gb-lcd/debug_render`: feature of a subcrate required by `debug_render`.

enabling `gb-lcd/debug_render` alone doesn't change much of the ui, and `debug_render` automatically enable it.
Maybe it is not necessary to mention `gb-lcd/debug_render` ?

Reply:
I've updated the table to show the requirement over `gb-lcd/debug_render` for `debug_render`